### DISCUSSION TITLE
Size of embedded cover art pictures

### DISCUSSION THREAD

**Discussion by kp-org** [jcorporation/myMPD#1139] [General]
If the cover art in the tags of an audio file is greater than 256kb it will not be shown in mympd. Initially I thought that png files will not be read, but I have had this behavior with jpg files of size > 256kB also. I'm not aware if there is a defined limit for embedded cover arts, but e.g. foobar doesn't have such a limit.

**Comment by jcorporation** [comment]
This should be fixed with 90383d1. The limit is documented here: https://jcorporation.github.io/myMPD/references/pictures
If this does not work for you with latest release please open a bug and use the bug report template.
